Packs/CommonScripts/Scripts/FormatURL/FormatURL.py

```diff
@@ -435,7 +453,7 @@ def check_valid_character(self, index: int) -> tuple[int, str]:
             raise URLError(f"Invalid character {self.modified_url[index]} at position {index}")
 
         else:
-            self.output += char
+            # self.output += char
```

Comment:
is this intentional? Why did you remove this part?

Reply:
Changed the way it works, instead of updating `self.output` and `part` it now only updates `part` and then adds it to output at the end of the check.

Reply:
Please remove the comment